config.toml

```diff
@@ -110,6 +110,19 @@ host = '127.0.0.1'
 # by the telemetry service. Default: 3001
 port = 3001
 
+[telemetry.buckets]
+# Specify the range of the 10 histogram buckets in ms for the `tx_latency_submitted` metric.
+# Default: { start = 500, end = 10000, buckets = 10 }
+# The default will give the following buckets:
+# [500, 1450, 2400, 3350, 4300, 5250, 6200, 7150, 8100, 9050, 10000]
+# latency_submitted = { start = 500, end = 10000, buckets = 10 }
```

Comment:
Was thinking about giving hints on the strategy for setting these values so I played a bit with this on a local testnet with block time of ~1sec and `batch_delay = 50ms`  trying to figure out the best start and end values. It's not very clear to me what the strategy is to set these values, seemed `end = 5s` worked best. Being also global configs they need to cover all chains that we relay on, so the chain block time cannot be used directly when reasoning about these values. Maybe taking the maximum block time and using a higher number of buckets so smaller block-time chains end up using 2-3 buckets.
Any thoughts on this?

Reply:
Good idea! So we would not ask for start/end but just the number of buckets and derive start/end from the highest max block time?

Reply:
The thing I dislike if we derive start/end from max block time is that it would no longer be possible to configure the metrics without changing a non-metric configuration.
I feel like the it should be possible to easily change the buckets in order to have additional information when something unexpected is happening.
I would be more inclined to adding some suggestion in the default `config.toml`, default values if start/end/buckets are not specified, and a more in depth explanation in the guide.